src/promptflow-core/promptflow/contracts/flow.py

```diff
@@ -1019,6 +1026,48 @@ def deserialize(data: dict) -> "FlexFlow":
             environment_variables=data.get("environment_variables") or {},
         )
 
+    @classmethod
+    def _from_dict(cls, flow_dag: dict, working_dir: Path, name=None) -> "FlexFlow":
```

Comment:
flow_dag should change as this is flex?

flow_dag-> flow_dict

Reply:
For the same function of dag flow and flex flow, I want to keep the parameters and names consistent, let me consider how to modify it.
![image](https://github.com/microsoft/promptflow/assets/23182548/fd5dda5b-7b3b-49a5-b5d6-e8d6e759534b)

Reply:
I will rename it to flow_data